package.json

```diff
@@ -114,21 +118,24 @@
 		"@skyra/saelem": "^3.0.2",
 		"@types/backoff": "^2.5.1",
 		"@types/diff": "^5.0.0",
+		"@types/google-protobuf": "^3.7.4",
 		"@types/he": "^1.1.1",
 		"@types/i18next-fs-backend": "^1.0.0",
-		"@types/ioredis": "^4.22.2",
+		"@types/ioredis": "^4.22.3",
```

Comment:
Revert lest this is going to cause conflicts down the road, i.e. if not merged before sunday and depfu comes around.